src/applications/vaos/reducers/appointments.js

```diff
@@ -183,6 +193,82 @@ export default function appointmentsReducer(state = initialState, action) {
         future: null,
         futureStatus: FETCH_STATUS.notStarted,
       };
+    case FETCH_EXPRESS_CARE_WINDOWS:
+      return {
+        ...state,
+        expressCare: {
+          ...initialState.expressCare,
+          fetchWindowsStatus: FETCH_STATUS.loading,
+        },
+      };
+    case FETCH_EXPRESS_CARE_WINDOWS_SUCCEEDED: {
+      const { facilityData } = action;
+      const now = moment.utc();
+
+      const times = facilityData
+        .reduce(function(arr, row) {
+          return arr.concat(row);
+        }, [])
+        .filter(f => !!f.expressTimes)
+        .map(f => {
+          const { expressTimes, authoritativeName, id } = f;
+          const { start, end, offsetUtc, timezone } = expressTimes;
+          const today = now.format('YYYY-MM-DD');
+          const startString = `${today}T${start}${offsetUtc}`;
+          const endString = `${today}T${end}${offsetUtc}`;
+
+          return {
+            utcStart: moment.utc(startString).format(),
+            utcEnd: moment.utc(endString).format(),
+            start: moment.parseZone(startString).format(),
+            end: moment.parseZone(endString).format(),
+            offset: offsetUtc,
+            timeZone: timezone,
+            name: f.authoritativeName,
+            id,
+          };
+        })
+        .sort((a, b) => (a.utcStart < b.utcStart ? -1 : 1));
+
+      let minStart;
+      let maxEnd;
+
+      if (times.length) {
+        const timesReverseSorted = times.sort(
+          (a, b) => (a.utcEnd > b.utcEnd ? -1 : 1),
+        );
+
+        minStart = times?.[0]?.utcStart;
+        maxEnd = timesReverseSorted?.[0]?.utcEnd;
+      }
+
+      return {
+        ...state,
+        expressCare: {
+          fetchWindowsStatus: FETCH_STATUS.succeeded,
+          allowRequests:
+            times.length &&
+            now.isAfter(moment.utc(minStart)) &&
+            now.isBefore(moment.utc(maxEnd)),
+          minStart,
+          maxEnd,
+          localWindowString:
+            minStart && maxEnd
+              ? `${moment(minStart).format('h:mm a')} to ${moment(
+                  maxEnd,
+                ).format('h:mm a')}`
+              : null,
```

Comment:
The original design displays the time zone, but a user can have systems in multiple time zones.  I think it makes the most sense here to display the window in the user's local time.

Reply:
I'm ambivalent about the time zone these are displayed in, but I think showing the timezone description is important, because we normally show things in facility local time.

Reply:
Yeah, I think I should have stuck with the idea of the "happy path" for the MVP.  We can decide how to handle the unlikely scenario multiple time zones with product/design in a different ticket possibly.  Will display the facility time zone.